web/pages/notifications.tsx

```diff
@@ -47,18 +112,51 @@ export default function Notifications() {
           className={'pb-2 pt-1 '}
           defaultIndex={0}
           tabs={[
+            {
+              title: 'New Notifications',
```

Comment:
Not a big fan of a separate tab for new vs. all notifications. Bad UX to have to click between them, plus New notifications are very ephemeral since they disappear once seen.

Isn't it standard to have like a colored dot on unseen notifications?

Reply:
I did it this way at a user's request for inbox status of 0 but I can see both sides.